src/comp.ts

```diff
@@ -120,7 +120,7 @@ export async function getCompBalance(
  */
 export async function getCompAccrued(
   _address: string,
-  _provider : Provider | string='mainnet'
+  _provider : Provider | string='rsk-testnet'
```

Comment:
```suggestion
  _provider : Provider | string='testnet'
```
Use same name as in constants